docs/dcp2_system_design.rst

```diff
@@ -646,16 +646,58 @@ Object names given in this section are relative to the staging area. To
 produce the complete ``gs://…`` URI of a particular object in the staging
 area, append the object's name to the staging area URI. [#]_
 
+
+Staging area properties
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Every staging area must include an object named ``staging_area.json`` in the
+root directory. It contains a JSON object that defines the *staging area
+properties* i.e. traits that apply to the staging area as a whole. Absence of
+this object must result in an immediate abort of the importer with an error
+message as defined in `Import errors`_. Legacy staging areas must be
+retrofitted with this object before they can be imported again.
+
+|nn| In this case, retrofitting seemed easier than defining defaults for each
+and every staging area property going forward. Similarly, as we add staging
+area  properties in the future, legacy staging areas would need to be
+retrofitted again with those new properties, but I feel it's a good thing that
+staging area sources are forced to make a concsious choice about the value of
+each new staging area property before their staging area can be imported
+again. |ne|
+
+The contents of ``staging_area.json`` must match the following schema::
+
+   {
+     "$schema": "https://json-schema.org/draft/2019-09/schema",
+     "properties": {
+       "is_delta": {
+         "type": "boolean"
+       }
+     },
+     "required": [
+       "is_delta"
+     ],
+     "additionalProperties": false
+   }
+
+The ``is_delta`` property indicates whether a staging contains exclusively
```

Comment:
Just to note, I believe we would have to stop using the project UUID as a staging name directory since we'd have to use a different staging area location for is_delta staging areas.

Reply:
> I believe we would have to stop using the project UUID

That should be 

> I believe we would have to stop using **just** the project UUID

Right?

Reply:
Addressed